### PR TITLE
Update os.sh

### PR DESCRIPTION
**Pull request by chadbrewbaker**
WSL detection support.

### CODE REVIEW

**Comment by robmorgan** [comment]
@chadbrewbaker looks okay to me. I don't run Windows or WSL but I wanted to check if they publish a `/etc/release` file?

File: modules/bash-commons/src/os.sh [hunk 1]
```diff
@@ -36,6 +36,10 @@ function os_is_redhat {
   grep -q "Red Hat Enterprise Linux Server release $version" /etc/*release
 }
 
+# Returns true (0) if this is a WSL kernel or false (1) otherwise.
+function os_is_wsl_kernel {
+  [[ $(uname -r | tail -c19) == "microsoft-standard" ]]
```
Comment (brikis98) [inline modules/bash-commons/src/os.sh:41]: How stable / reliable is looking at column 19? Could that change in a future release?

Comment (infraredgirl) [reply modules/bash-commons/src/os.sh:41]: This does seem brittle... Maybe use `grep` instead?

Comment (chadbrewbaker) [reply modules/bash-commons/src/os.sh:41]: Bash supports regex, no need to grep, /etc/os-release would be Ubuntu/Redhat etc, whatever WSL2 container the user has loaded. It's a config string in their build, https://github.com/microsoft/WSL2-Linux-Kernel/blob/master/Microsoft/config-wsl#L22

**Pull request closed without merging**